pkg/operation/operation.go

```diff
@@ -502,30 +500,49 @@ func (o *Operation) InjectShootShootImages(values map[string]interface{}, names
 	return chart.InjectImages(values, o.ImageVector, names, imagevector.RuntimeVersion(o.ShootVersion()), imagevector.TargetVersion(o.ShootVersion()))
 }
 
-// EnsureShootStateExists creates the ShootState resource for the corresponding shoot and sets its ownerReferences to the Shoot.
+// EnsureShootStateExists creates the ShootState resource for the corresponding shoot and updates the operations object
 func (o *Operation) EnsureShootStateExists(ctx context.Context) error {
+	var (
+		err        error
+		shootState = &gardencorev1alpha1.ShootState{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      o.Shoot.Info.Name,
+				Namespace: o.Shoot.Info.Namespace,
+			},
+		}
+	)
+
+	if err = o.K8sGardenClient.Client().Create(ctx, shootState); err != nil {
```

Comment:
Probably switch the order of `Get` and `Create`, as `Get` happens a lot more frequently?

Reply:
This won't work as the seed authorizer won't allow a `Get` if the object does not exist yet.
Let's
/hold
this PR until #4044 is merged where this function is adapted, see https://github.com/gardener/gardener/pull/4044/files#diff-1b66a0fabd3ab9fcedbaaa435a65b221b2208c2874ce24f07692152e2e32a574L530.

Reply:
Ah right. Forgot about that again, sorry 🙈